WalletWasabi.Fluent/ViewModels/Wallets/Send/ChangeAvoidanceSuggestionViewModel.cs

```diff
@@ -124,14 +147,19 @@ public partial class ChangeAvoidanceSuggestionViewModel : SuggestionViewModel
 			suggestions.Add(smallerSuggestion);
 		}
 
+		if (bnbSuggestion is { })
```

Comment:
For new code, I would suggest:

```suggestion
		if (bnbSuggestion is not null)
```